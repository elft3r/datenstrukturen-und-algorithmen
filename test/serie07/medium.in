10
15 100
936 798 733 324 345 652 918 720 905 784 683 124 340 488 304
17 8 8 18 4 19 11 11 8 3 7 17 11 11 8 
14 84
946 623 366 299 320 850 650 201 637 738 430 312 30 645
4 20 4 7 12 12 14 18 6 18 20 18 19 9 
18 81
271 921 503 697 464 390 928 671 574 896 263 459 114 946 488 475 477 545
4 12 18 12 7 3 18 9 10 18 18 20 4 7 19 16 19 8 
16 72
320 273 494 420 814 961 886 856 98 60 375 185 20 445 43 833
13 9 17 13 8 15 13 6 18 12 11 18 2 2 2 14 
14 100
514 958 655 921 221 750 812 456 277 785 656 301 4 776
7 14 8 6 17 15 11 16 11 15 7 16 11 17 
12 46
505 639 902 489 159 95 781 774 555 443 863 944
7 9 10 13 14 9 3 4 12 10 10 8 
18 68
656 92 798 706 355 905 301 588 602 907 536 712 998 805 822 817 601 426
19 16 15 6 10 12 5 4 14 12 6 4 1 11 20 9 9 7 
17 102
238 448 911 228 977 688 407 93 163 597 847 664 112 1 647 902 429
3 12 3 19 14 10 7 10 4 13 7 8 12 14 15 6 9 
15 53
396 656 875 117 888 944 949 825 56 554 391 508 655 104 63
11 5 1 3 15 1 16 6 7 10 15 15 19 11 15 
18 91
775 960 925 616 508 644 42 149 360 652 854 28 233 796 146 606 63 490
2 14 15 19 8 9 1 16 3 14 10 14 18 6 5 19 18 1 
