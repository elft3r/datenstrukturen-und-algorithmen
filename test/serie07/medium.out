7650 1 2 3 5 7 8 9 10 11 13 14
5152 1 3 5 6 7 9 10 14
5833 1 2 4 5 6 7 8 14 18
5215 4 5 6 7 8 14 16
6827 1 2 3 4 6 7 10 11 14
4903 2 3 7 8 11 12
7071 4 6 7 8 10 11 12 13 16 17
7489 1 2 3 5 6 7 9 10 11 12 15 16
5316 1 2 3 4 6 7 8 10
7152 1 2 3 5 6 7 9 10 11 14 15 18
